python/torch_mlir/dialects/torch/importer/jit_ir/csrc/node_importer.cpp

```diff
@@ -233,6 +233,13 @@ void NodeImporter::importNode(Node *node, MlirBlock appendToBlock,
                        dummyAnnotator, importOptions);
       mapResults(node, mlirOpResultGetOwner(listValue));
       return; // Early return, since `importIValue` already added op to block.
+    } else if (output->type()->cast<c10::TupleType>()) {
```

Comment:
This is the same code as the `ListType` case above. We should just change the `if` condition and reuse the logic.